camel-workitem/src/main/java/org/jbpm/process/workitem/camel/CXFCamelWorkitemHandler.java

```diff
@@ -48,7 +49,8 @@
         },
         serviceInfo = @WidService(category = "${name}", description = "${description}",
                 keywords = "apache,camel,cxf,payload,route,connector",
-                action = @WidAction(title = "Connect to a JAX-WS service hosted in CXF")
+                action = @WidAction(title = "Connect to a JAX-WS service hosted in CXF"),
+                authinfo = @WidAuth
```

Comment:
Why is this needed ? Looks weird

Reply:
@tsurdilo Please see the discussion in GChat.

Reply:
I have lots of messages in GChat, also other chats. Could you maybe give a summary here. I can also do all my PR reviews on GChat as well if this is the way we are going?

Reply:
@tsurdilo Sorry, but the original intention was to point you to the whole thread we already have there about this topic...

Here is the copied summary:

When compiling with JDK 11 to JDK 8 target it fails with the message:

`Error post-processing workitem annotations: Incorrectly typed data found for annotation element public abstract org.jbpm.process.workitem.core.util.service.WidAuth org.jbpm.process.workitem.core.util.service.WidService.authinfo() (Found data of type org.jbpm.process.workitem.core.util.service.WidAuth)`

The issue is WidAuth String[] parameters (array of strings) - when I remove them it works 🙂 Also if I compile to JDK 11 target it works. In fact, it is even enough to add an import of WidAuth into the annotated file (WIH in this case) and it will start working. But in that way the import is redundant (and could be easily removed by mistake), so I decided to explicitly define @WidAuth which will solve this issue and there will be no effect on the code. I already have a reproducer which I will use in the issue I want to file in maven-compiler-plugin and if it gets forwarded to Java compiler, I will create an issue in Java compiler. Hope that helped.

Reply:
Thanks!! I guess this is fine then if otherwise does not compile.

Reply:
Yep, the issue is really weird, but it works now :)